client/src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import { Event } from "./components/Event/Event";
+import { events } from "./components/Event/data/data";
+
+export default function App() {
+    return (
+        <div>
+            {events.map((event) =>(
+            <Event key={event.id} lokace={event} nazev={event} datum={event} />
```

Comment:
tohle neni uplne dobre
```
lokace={event} nazev={event} datum={event}
```

do 3 ruznych props predavas to same (a pak to diky tomu spatne pouzivas v Event komponente. Melo by to byt napr.

```
<Event key={event.id} lokace={event.location} nazev={event.title} datum={event.dates} />
```

a nalezite k tomu zmenit komponentu Event